cli/compose/loader/loader.go

```diff
@@ -32,6 +32,8 @@ type Options struct {
 	SkipInterpolation bool
 	// Interpolation options
 	Interpolate *interp.Options
+	// Remove 'env_file' entries after resolving to 'environment' section
+	RemoveEnvFiles bool
```

Comment:
Looking at this again; wondering if `DiscardEnvFiles` or `ClearEnvFiles` would be less confusing. `Remove....` may sound as if the actual files are removed?

(open to suggestions)